### PR TITLE
Move study and splash effects assets to package

### PR DESCRIPTION
**Pull request by guidezpl**
Part of work for #87 

Updated https://pub.dev/packages/flutter_gallery_assets/versions/0.2.0

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -43,18 +43,59 @@ flutter:
...
     - assets/icons/settings/
     - assets/logo/
-    - assets/splash_effects/
-    - packages/rally_assets/logo.png
-    - packages/rally_assets/thumb.png
-    - assets/crane/logo/
-    - assets/crane/destinations/
-    - assets/fortnightly/
+    - assets/studies/
+    - packages/flutter_gallery_assets/crane/destinations/eat_1.jpg
```
Comment (rami-a) [inline pubspec.yaml:52]: It's annoying that when you use a package, you have to specify each asset individually. But oh well

File: pubspec.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ dependencies:
   url_launcher: ^5.1.6
   shared_preferences: ^0.5.4
   collection: ^1.14.0
-  flutter_gallery_assets: ^0.1.9
+  flutter_gallery_assets: ^0.2.0
```
Comment (rami-a) [inline pubspec.yaml:27]: Wouldn't the old assets still exist in the git history of the repo, so would the repo size actually be reduced?

File: pubspec.yaml (hunk 1 above)
Comment (guidezpl) [reply pubspec.yaml:52]: I'm working on a change to allow the same directory specifying as in local assets, just need more time to get it done :)

File: pubspec.yaml (hunk 2 above)
Comment (guidezpl) [reply pubspec.yaml:27]: My next step as part of #87 is to rewrite the history to scrap all deleted files

**Pull request merged**